addons/l10n_ae/models/account_move_line.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models, fields, api
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    tax_ids = fields.Many2many(string='VAT')
```

Comment:
tax_ids in other cases (localizations, ...) will not only be VAT.  (maybe there are cases in UAE as well)

Reply:
That's a really good point. How can we make sure to set the label of all tax_ids to 'VAT' when the company is based in UAE (assuming a multi company setting). For us in UAE we only have the VAT.

Reply:
Any suggestions regarding this point?

Reply:
Remove this and change from view